src/app/static/src/tests/components/runReport/parameterList.test.ts

```diff
@@ -0,0 +1,42 @@
+import {shallowMount} from "@vue/test-utils";
+import ParameterList from "../../../js/components/runReport/parameterList.vue"
+
+
+describe(`run report parameter list`, () => {
+    const params = [
+        {name: "global", value: "initial value"},
+        {name: "minimal", value: "initial value 2"}
+    ]
+
+    const store = () => shallowMount(ParameterList,
+        {
+            propsData: {params: params}
+        }
+    )
+
+    it(`can render parameter labels, values and props as expected`, () => {
+        const wrapper = store()
+        const labels = wrapper.find("table").findAll("label")
+
+        expect(labels.at(0).text()).toBe("global")
+        expect(labels.at(1).text()).toBe("minimal")
+        expect(wrapper.vm.$data.paramValues).toBe(params)
```

Comment:
Checking `$data` is correct isn't the same as ensuring the the table is populated correctly. The following would be ideal:
```suggestion
        const inputs = wrapper.find("table").findAll("input")
        const globalInput = inputs.at(0).element as HTMLInputElement
        expect(globalInput.value).toBe("initial value")
        const minimalInput = inputs.at(1).element as HTMLInputElement
        expect(minimalInput.value).toBe("initial value 2")
```

Reply:
This suggestion is useful.